pylot/utils.py

```diff
@@ -538,20 +538,21 @@ class CanBus(object):
             vehicle.
         forward_speed (:obj:`int`): Forward speed in m/s.
     """
-    def __init__(self, transform, forward_speed):
+    def __init__(self, transform, forward_speed, velocity_vector=None):
         if not isinstance(transform, Transform):
             raise ValueError(
                 'transform should be of type pylot.utils.Transform')
         self.transform = transform
         # Forward speed in m/s.
         self.forward_speed = forward_speed
+        self.velocity_vector = velocity_vector
```

Comment:
Can you please extend the documentation to include velocity_vector. Is it in world frame or vehicle reference frame?

Reply:
Done, it's in world frame.